### ISSUE TITLE
Suggestion: Add More Names For Config

### ISSUE THREAD

**Issue by RocketManKian** [DiscordSRV/DiscordSRV#954]
Hi there! So today I discovered that the Channel Topic Updater will only work and post in the first channel mentioned in the config.yml which is my global channel, so I decided to swap this round and put my main channel as the first listed one so it would post in there which is all great. But then I realised the Server Start and Server Close both also close in the first mentioned channel. This is causing my Server Start and Server End notifications to post in the channel I don't intend for it to post in

So basically my suggestion is to add channel names for server start and server end just like there is for join and leave.

Thanks!

**Comment by Dinty1** [comment]
This has been implemented in testing builds for v2 of DiscordSRV (Ascension). See [here](https://github.com/DiscordSRV/Ascension?tab=readme-ov-file#ascension) for more info.
